### ISSUE TITLE
CF applied to the name of the film

### ISSUE THREAD

**Issue by Tamberlox** [Radarr/Radarr#7004]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When importing this release, radarr applies both my [Tigole] and [Silence] CF naming formats even though Silence is the name of the film.

So the release `The Silence of the Lambs (1991) Criterion (1080p BluRay x265 HEVC 10bit AAC 5.1 Tigole) [QxR]`

becomes `The Silence of the Lambs (1991) - [Bluray-1080p] [AAC 5.1] [x265 10bit] [Tigole Silence] {imdb-tt0102926}.mkv`

### Expected Behavior

The release `The Silence of the Lambs (1991) Criterion (1080p BluRay x265 HEVC 10bit AAC 5.1 Tigole) [QxR]`

should become `The Silence of the Lambs (1991) - [Bluray-1080p] [AAC 5.1] [x265 10bit] [Tigole] {imdb-tt0102926}.mkv` after import

### Steps To Reproduce

1. Set film format to {Movie CleanTitle} ({Release Year}) - {[Edition Tags]} {[Quality Full]} {[Mediainfo AudioCodec} { Mediainfo AudioChannels}] [{Mediainfo VideoCodec} {Mediainfo VideoBitDepth}bit] [{Custom Formats}] {imdb-{ImdbId}}
2. Download the same release
3. Let import process begin
4. See result

### Environment

```markdown
- OS: DSM 7.0.1
- Radarr:4.0.4.5896
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: Microsoft Edge
```

### What branch are you running?

Nightly

### Trace Logs?

Not sure how to get them

**Comment by foux** [comment]
If you're CF has "Silence" in the release title, then I don't see any bug. But you'll have to show us your CF

**Comment by Drapersniper** [comment]
Do you have the option to add the CF to the name ticked under the specific CF ? If so this is not so much a bug, more of a feature request.

**Comment by bakerboy448** [comment]
What is the custom format used, but this does not appear to be a bug if "Silence" is looked for depending on the specific CF

**Comment by ta264** [comment]
I disagree, we explicitly try to strip out the movie name when matching CF against an indexer release, but it appears we don't do the same when matching against files on disk. The two should at least be consistent in my opinion.

**Comment by rg9400** [comment]
Bumping this as I've noticed it being a cause for download loops. As ta mentioned, this will just bring consistency to the logic being used when grabbing a release (parsing releasetitle) and importing (parsing filename). It would also be nice to see this filter downstream to Sonarr for SeriesTitle and EpisodeTitle as well. This would make a lot of false positives with CFs disappear and give more control when creating them.
